include/boost/http_proto/server/helmet.hpp

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (c) 2025 Amlal El Mahrouss (amlal at nekernel dot org)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/cppalliance/http_proto
+//
+
+#ifndef BOOST_HTTP_PROTO_SERVER_HELMET_HPP
+#define BOOST_HTTP_PROTO_SERVER_HELMET_HPP
+
+#include <boost/http_proto/detail/config.hpp>
+#include <boost/http_proto/server/route_handler.hpp>
+#include <memory>
+
+namespace boost {
+namespace http_proto {
+
+/// @brief Helmet options.
+struct options
```

Comment:
you think `http_proto::options` is a good name for something that is helmet specific? What do we do with options for the other middlewares?